### PR TITLE
Merge DBCParser into Mapper

### PR DESCRIPTION
**Pull request by sophokles73**
The readers require a Mapper as well as information provided by the DBCParser. The Mapper itself also requires information from the parser.

The Mapper has therefore been changed to use the DBCParser class as a mixin. The readers and the Feeder have been adapted accordingly.

### CODE REVIEW

File: dbc2val/dbcfeederlib/dbc2vssmapper.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python3
 
 ########################################################################
-# Copyright (c) 2020 Robert Bosch GmbH
+# Copyright (c) 2020,2023 Robert Bosch GmbH
```
Comment (erikbosch) [inline dbc2val/dbcfeederlib/dbc2vssmapper.py:4]: @SebastianSchildt - do we somewhere have any guidelines on how to handle "years" in copyright? As of today we have for dbcfeeder two types of license statements, one (like here) stating Bosch copyright, others have a more generic "Contributors to the Eclipse Foundation". Then for year, when shall we add a new year like here. Always?

Comment (sophokles73) [reply dbc2val/dbcfeederlib/dbc2vssmapper.py]: The same section also states for the newer copyright header's 1st line:

> The year is expressed as a single year or a range spanning the year of the very initial assertion and that of the most recent change.

:-)

File: dbc2val/test/test_j1939/test_j1939.py [hunk 2]
```diff
@@ -1,72 +0,0 @@
-#!/usr/bin/python3
-
-########################################################################
-# Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (erikbosch) [inline dbc2val/test/test_j1939/test_j1939.py:4]: Intentional to remove this file? - it was just added, right?

Comment (sophokles73) [reply dbc2val/test/test_j1939/test_j1939.py:4]: it's the latter :-)

**Review by erikbosch — APPROVED** [review]

**Pull request merged**